migrations/2_deploy_contracts.js

```diff
@@ -13,10 +13,27 @@ const fs = require("fs");
 const config = require("../deploy.config");
 const providerConfig = require("../providers.config");
 
+// used for amm strategies
+const tokenIndexMapping = {
```

Comment:
don't we need to add mobius strategies?

Reply:
No we don't need there at least for the current pools

Reply:
you mean not for the current pools because they use 0, or because they don't use token index?

Reply:
Current pools the cusd index is 0 so hence but I'll add it going forward